cmd/waku/server/rest/waku_rest.go

```diff
@@ -18,10 +18,11 @@ type WakuRest struct {
 
 	log *zap.Logger
 
-	relayService *RelayService
+	relayService  *RelayService
+	filterService *FilterService
 }
 
-func NewWakuRest(node *node.WakuNode, address string, port int, enablePProf bool, enableAdmin bool, relayCacheCapacity int, log *zap.Logger) *WakuRest {
+func NewWakuRest(node *node.WakuNode, address string, port int, enablePProf bool, enableAdmin bool, relayCacheCapacity, filterCacheCapacity int, log *zap.Logger) *WakuRest {
```

Comment:
I'm thinking it's time to refactor this function to receive a struct with the configuration instead of adding more functions! I'll open an issue so we can do that on a separate PR!

Reply:
Tracking it here https://github.com/waku-org/go-waku/issues/859